qiskit/test/mock/fake_openpulse_2q.py

```diff
@@ -58,6 +58,8 @@ def __init__(self):
             ],
             meas_kernels=['kernel1'],
             discriminators=['max_1Q_fidelity'],
+            default_meas_kernel='kernel1',
```

Comment:
might as well make these `hw_boxcar` and `linear_discriminator`.

Reply:
These are tests, so it really doesn't matter. And I don't want to change it if we change some of the default discriminators. But I will change it for now.

Reply:
Addressed in https://github.com/Qiskit/qiskit-terra/pull/5131/commits/2bb40181110801931b8c77e7e114be2cfeeeca9f.